src/java.base/share/classes/java/io/SimpleIO.java

```diff
@@ -0,0 +1,1037 @@
+                return scanner.nextBoolean();
+            }
+        }
+    }
+
+    private static final Object[] NULL_ARRAY = new Object[1];
+
+    /**
+     * Single space print values to the output stream.
+     * For example:
+     * {@snippet lang="java":
+     * print("A", "B"); // @highlight substring="print"
+     * print("C", "D"); // @highlight substring="print"
+     * }
+     * will print on the output stream as:
+     * {@snippet lang="text":
+     * A BC D
+     * }
+     *
+     * @param values values to be printed.
+     */
+    public static void print(Object... values) {
+        System.out.print(Stream.of(values == null ? NULL_ARRAY : values)
+                .map(String::valueOf)
+                .collect(Collectors.joining(" ")));
+    }
+
+    /**
+     * Single space print values to the output stream followed by the
+     * platform line terminator.
+     * For example:
+     * {@snippet lang="java":
+     * printLine("A", "B"); // @highlight substring="printLine"
+     * printLine("C", "D"); // @highlight substring="printLine"
+     * }
+     * will print on the output stream as:
+     * {@snippet lang="text":
+     * A B
+     * C D
+     *
+     * }
+     *
+     * @param values values to be printed.
+     */
+    public static void printLine(Object... values) {
+        System.out.println(Stream.of(values == null ? NULL_ARRAY : values)
+                .map(String::valueOf)
+                .collect(Collectors.joining(" ")));
+    }
+
+    /**
+     * Print list of lines to output stream. Each line will be printed and
+     * followed by a line terminator.
+     * For example:
+     * {@snippet lang="java":
+     * var fruit = List.of("apple", "pear", "orange");
+     * printLines(fruit); // @highlight substring="printLines"
+     * }
+     * will print on the output stream as:
+     * {@snippet lang="text":
+     * apple
+     * pear
+     * orange
+     *
+     * }
+     *
+     * @param lines list of strings
+     * @throws NullPointerException if lines is null
+     */
+    public static void printLines(List<String> lines) {
+        Objects.requireNonNull(lines, "lines must not be null");
+        lines.forEach(System.out::println);
+    }
+
+    /**
+     * The contents of the file {@code filename} are read as a string.
+     * For example:
+     * {@snippet lang="java":
+     * var text = """
+     *    "Hope" is the thing with feathers
+     *    That perches in the soul
+     *    And sings the tune without the words
+     *    And never stops - at all
+     *    """;
+     * write("data.txt", text);
+     * var data = read("data.txt"); // @highlight substring="read"
+     * print(data);
+     * }
+     * will print on the output stream as:
+     * {@snippet lang="text":
+     * "Hope" is the thing with feathers
+     * That perches in the soul
+     * And sings the tune without the words
+     * And never stops - at all
+     *
+     * }
+     *
+     * @param filename  file name of file to be read
```

Comment:
The audience may be new developers but "file name" could be mislead as it can be a full file path, might have to tweak this a bit to get right.

Reply:
Thx